apps/functions-subscription/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "functions-subscription",
+  "private": true,
+  "main": "./dist/main.js",
+  "version": "0.0.0",
+  "description": "The Azure function that let users to subscribe to a trial and to provide information about the subscriptions.",
+  "devDependencies": {
+    "@pagopa/openapi-codegen-ts": "^13.1.0",
+    "@tsconfig/node20": "^20.1.4",
+    "azure-functions-core-tools": "^4.0.5700",
+    "eslint": "^9.2.0",
+    "eslint-config-rules": "workspace:^",
+    "rimraf": "^5.0.5",
+    "shx": "^0.3.4",
+    "typescript": "^5.4.5"
+  },
+  "dependencies": {
+    "@azure/functions": "^4.4.0",
+    "@pagopa/handler-kit": "^1.1.0",
+    "@pagopa/handler-kit-azure-func": "^2.0.4",
+    "@pagopa/io-functions-commons": "^29.0.0",
+    "@pagopa/ts-commons": "^13.0.0",
+    "fp-ts": "^2.13.1",
+    "io-ts": "^2.2.19"
+  },
+  "scripts": {
+    "clean": "rimraf ./dist ./src/generated/definitions/internal",
+    "build": "tsc",
+    "lint": "eslint --fix",
+    "lint:check": "eslint",
+    "generate": "rimraf ./src/generated/definitions/internal && shx mkdir -p ./src/generated/definitions/internal && gen-api-models --api-spec ./api/internal.yaml --no-strict --out-dir ./src/generated/definitions/internal",
+    "start": "func start --javascript"
+  }
```

Comment:
**suggestion (non-blocking)**: Reorder scripts, dependencies and devDependencies 🤓. Moreover I've add the typecheck command

```suggestion
  "scripts": {
    "clean": "rimraf ./dist ./src/generated/definitions/internal",
    "typecheck": "yarn build --noEmit",
    "build": "tsc",
    "lint": "eslint --fix",
    "lint:check": "eslint",
    "generate": "rimraf ./src/generated/definitions/internal && shx mkdir -p ./src/generated/definitions/internal && gen-api-models --api-spec ./api/internal.yaml --no-strict --out-dir ./src/generated/definitions/internal",
    "start": "func start --javascript"
  },
  "dependencies": {
    "@azure/functions": "^4.4.0",
    "@pagopa/handler-kit": "^1.1.0",
    "@pagopa/handler-kit-azure-func": "^2.0.4",
    "@pagopa/io-functions-commons": "^29.0.0",
    "@pagopa/ts-commons": "^13.0.0",
    "fp-ts": "^2.13.1",
    "io-ts": "^2.2.19"
  },
    "devDependencies": {
    "@pagopa/openapi-codegen-ts": "^13.1.0",
    "@tsconfig/node20": "^20.1.4",
    "azure-functions-core-tools": "^4.0.5700",
    "eslint": "^9.2.0",
    "eslint-config-rules": "workspace:^",
    "rimraf": "^5.0.5",
    "shx": "^0.3.4",
    "typescript": "^5.4.5"
  }
```

Reply:
Done!